libraries/aws_cloudfront_distribution.rb

```diff
@@ -44,6 +44,9 @@ def initialize(opts = {})
     # The Viewer Certificate minimum SSL version at the time of writing is TLSv1 for a default cloud certificate,
     # and for custom certifcates one of TLSv1.2_2021, TLSv1.2_2019, TLSv1.2_2018, TLSv1.1_2016, TLSv1_2016 or TLSv1
     @viewer_certificate_minimum_ssl_protocol = config.viewer_certificate[:minimum_protocol_version]
+    
+    # get origin path
+    @s3_origin_path = [config.origins.items.origin_path]
```

Comment:
items is an array. So I think it should be "config.origins.items.first.origin_path"

Or you have to add a method to support this.

Reply:
https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/CloudFront/Client.html#get_distribution-instance_method

Reply:
We are getting the exception "undefined method `origin_path' for #<Array:0x00007fafd1ef6ae0>" when we are trying to run a inspec test.

describe aws_cloudfront_distribution('cloudfront-1'') do
  it { should exist }
end

Reply:
checking

Reply:
There is another approach:

@distribution_config_origins = config.origins.to_h
create_resource_methods(@distribution_config_origins)

and below the initialize block add another function:
def origin_path
    items.map(&:origin_path)
end

In the inspec tests we can directly use the origin_path'
describe aws_cloudfront_distribution('sample') do
  its('origin_path') { should eq 'sample_path' }
end